src/dmclock_server.h

```diff
@@ -744,6 +764,14 @@ namespace crimson {
       }
 
 
+      inline const ClientInfo get_cli_info(ClientRec& client) const {
+	if (dynamic_cli_info_f) {
```

Comment:
I had originally intended to use well-named variable to support complie time constant, as you said.
The reason I used bool variable was to change the use of dynamic_cli_info_f via the configuration change even after the queue was created.
However, it seems that it is not necessary now.